src/input/mpegts.h

```diff
@@ -355,6 +355,7 @@ enum mpegts_mux_epg_flag
   MM_EPG_ONLY_UK_FREESAT,
   MM_EPG_ONLY_UK_FREEVIEW,
   MM_EPG_ONLY_VIASAT_BALTIC,
+  MM_EPG_ONLY_BULSATCOM_39E,
```

Comment:
This break compatibility - new values must be last, because these values are used in config files, too.